src/intersection-observer-polyfill.js

```diff
@@ -453,9 +435,9 @@ function calculateChangeEntry(
   return /** @type {!IntersectionObserverEntry} */ ({
     time: (typeof performance !== 'undefined' && performance.now) ?
         performance.now() : Date.now() - INIT_TIME,
-    rootBounds: rootBounds && DomRectFromLayoutRect(rootBounds),
-    boundingClientRect: DomRectFromLayoutRect(boundingClientRect),
-    intersectionRect: DomRectFromLayoutRect(intersection),
+    rootBounds: rootBounds && rootBounds,
```

Comment:
return `rootBounds` here is enough since we don't need to check `rootBounds == null`  due to `DOMRectFromLayoutRect` anymore.